packages/web/src/contexts/ModalsContext.js

```diff
@@ -75,12 +76,13 @@ const ModalsContextProvider = props => {
 	const [state, dispatch] = useReducer(reducer, { ...INITIAL_STATE })
 
 	const closeModal = useCallback(modalName => {
+		console.log(modalName)
```

Comment:
👀

Reply:
Surprised I only left one console.log in here.

Do you have a suggestion for other debugging tools? I have React dev tools which is great and i should probably just use that more.

Reply:
Nah, I use `console.log` a lot, too. Theoretically you can set it up so that Next exposes the Node debugger, then you can connect to that with VS Code and debug straight from your IDE. I always have issues when I try to set that up, though.